plugins/rapid7_intsights/icon_rapid7_intsights/util/api.py

```diff
@@ -38,10 +38,10 @@ def to_dict(self) -> dict:
                 "sourceDateTo": int(self.source_date_to) if self.source_date_to else None,
                 "foundDateFrom": int(self.found_date_from) if self.found_date_from else None,
                 "foundDateTo": int(self.found_date_to) if self.found_date_to else None,
-                "assigned": self.assigned == "Assigned" if self.assigned else None,
-                "isFlagged": self.is_flagged == "Flagged" if self.is_flagged else None,
-                "isClosed": self.is_closed == "Closed" if self.is_closed else None,
-                "hasIoc": self.has_ioc,
+                "assigned": json.dumps(self.assigned == "Assigned" if self.assigned else None),
+                "isFlagged": json.dumps(self.is_flagged == "Flagged" if self.is_flagged else None),
+                "isClosed": json.dumps(self.is_closed == "Closed" if self.is_closed else None),
+                "hasIoc": json.dumps(self.has_ioc if self.has_ioc else None),
```

Comment:
Please add unit-tests to reflect that change

Reply:
When I ran `coverage run -m unittest *` & `coverage report -m` it seems to be that those lines are already covered by the existing unit tests